Uppsala/UppEt1.xml

```diff
@@ -0,0 +1,470 @@
+                            </msItem>
+                                
+                            <msItem xml:id="ms_i1.8">
+                                <locus from="265rb" to="271vb"/>
+                                <title type="complete" ref="LIT3086RepCh366"/>
+                            </msItem>
+                            
+                        </msItem>
+                    </msContents>
+                    
+
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">iii + 273</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>160</height>
+                                        <width>140</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="14 16">
+                                    <locus from="ir" to="230"></locus>
+                                </layout>
+                                <layout columns="2" writtenLines="14 16">
+                                    <locus from="231" to="273"></locus>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <locus from="1r" to="64v"/>
+                                <locus from="151r" to="214v"/>
+                                <desc>According to <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">35</citedRange></bibl>,
+                                    the parts produced by <ref type="hand" corresp="#h1"></ref> ("A") are 
+                                    "very beautifully written" (<foreign xml:lang="de">sehr schön geschrieben</foreign>). 
+                                    <bibl><ptr target="bm:Loefgren1974Katalog"/></bibl> dates the hand to the <date notBefore="1500" notAfter="1599">16th century</date>,
+                                    whereas
+                                    <bibl>
+                                        <ptr target="bm:Zettersteen1899Upsala"/>
+                                        <citedRange unit="page">509</citedRange>
+                                    </bibl>
+                                    dates it to the <date notBefore="1400" notAfter="1499">15th century</date>.</desc>
+                                <date notBefore="1500" notAfter="1599">16th century.</date>
```

Comment:
seems like something wrong.

Reply:
The first date is within the description (and refers to Zetterstéen's dating). The second is the actual dating (based on Löfgren)